Broken builds from ppv-lite86 0.2.11

Hello,

Our builds started failing when version 0.2.11 of ppv-lite86 was published to crates.io.

We are seeing failures such as:
```
    --> /usr/rust/cargo/registry/src/github.com-1ecc6299db9ec823/ppv-lite86-0.2.11/src/x86_64/sse2.rs:1481:28
     |
1481 |                 let ab00 = _mm256_permute2x128_si256::<0x20>(a.x[0], b.x[0]);
     |                            ^^^^^^^^^^^^^^^^^^^^^^^^^-------- help: remove these generics
     |                            |
     |                            expected 0 const arguments
```

```
 error[E0061]: this function takes 3 arguments but 2 arguments were supplied
    --> /usr/rust/cargo/registry/src/github.com-1ecc6299db9ec823/ppv-lite86-0.2.11/src/x86_64/sse2.rs:1484:28
     |
1484 |                 let ab11 = _mm256_permute2x128_si256::<0x31>(a.x[1], b.x[1]);
     |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ ------  ------ supplied 2 arguments
     |                            |
     |                            expected 3 arguments
```

Can you please yank this release? These changes seem to not be compatible with a semver point release.

--- a/utils-simd/ppv-lite86/src/x86_64/sse2.rs
+++ b/utils-simd/ppv-lite86/src/x86_64/sse2.rs
@@ -1478,14 +1478,14 @@ pub mod avx2 {
              * a11:b11 c11:d11
              */
             unsafe {
-                let ab00 = _mm256_permute2x128_si256::<0x20>(a.x[0], b.x[0]);
-                let ab01 = _mm256_permute2x128_si256::<0x31>(a.x[0], b.x[0]);
-                let ab10 = _mm256_permute2x128_si256::<0x20>(a.x[1], b.x[1]);
-                let ab11 = _mm256_permute2x128_si256::<0x31>(a.x[1], b.x[1]);
-                let cd00 = _mm256_permute2x128_si256::<0x20>(c.x[0], d.x[0]);
-                let cd01 = _mm256_permute2x128_si256::<0x31>(c.x[0], d.x[0]);
-                let cd10 = _mm256_permute2x128_si256::<0x20>(c.x[1], d.x[1]);
-                let cd11 = _mm256_permute2x128_si256::<0x31>(c.x[1], d.x[1]);
+                let ab00 = _mm256_permute2x128_si256(a.x[0], b.x[0], 0x20);
+                let ab01 = _mm256_permute2x128_si256(a.x[0], b.x[0], 0x31);
+                let ab10 = _mm256_permute2x128_si256(a.x[1], b.x[1], 0x20);
+                let ab11 = _mm256_permute2x128_si256(a.x[1], b.x[1], 0x31);
+                let cd00 = _mm256_permute2x128_si256(c.x[0], d.x[0], 0x20);
+                let cd01 = _mm256_permute2x128_si256(c.x[0], d.x[0], 0x31);
+                let cd10 = _mm256_permute2x128_si256(c.x[1], d.x[1], 0x20);
+                let cd11 = _mm256_permute2x128_si256(c.x[1], d.x[1], 0x31);
                 (
                     Self { x: [ab00, cd00], ni: a.ni },
                     Self { x: [ab01, cd01], ni: a.ni },

